test/storeFile.m

```diff
@@ -0,0 +1,17 @@
+function storeFile(filepath, filename)
```

Comment:
Please include a header here describing what this function is doing and what is expected as input.